src/targets/resource.js

```diff
@@ -0,0 +1,33 @@
+import { ProgressBar } from "../ui/progress-bar.js";
+import { CONSTANTS, calcLevel } from "../constants/constants.js";
+import { Target } from "./target.js";
+
+export class Resource extends Target {
+    skill;
```

Comment:
What is this lol

Reply:
Oh this is how you declare globals in js? Gross

Reply:
Do you mean export or what? Export just means other files can import that var/function/class. So sorta?

Reply:
No just saying "skill;". I'm assuming that's how you declare a global without initializing in js

Reply:
That's one way to declare a class member variable, it's not global. You can technically declare and use by just doing `this.varName = 0` and then using it somewhere, but I do it this way to show more clearly what vars we're using

Reply:
Or I could do `skill = {};` or `skill = "";` of course